config/settings/base.py

```diff
@@ -260,6 +265,14 @@
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-soft-time-limit
 # TODO: set to whatever value is adequate in your circumstances
 CELERYD_TASK_SOFT_TIME_LIMIT = 60
+
+CELERY_BEAT_SCHEDULE = {
+    "clean_stale_data": {
+        "task": "dear_petition.petition.tasks.clean_stale_data.clean_stale_data_task",
+        "schedule": crontab(minute=0, hour="2"),
```

Comment:
Does this run every day at 2 AM? Can you add a comment to indicate when it's expected to run?

Reply:
Sure. Yes, this schedules it to run at 2 AM EST each night.